client/app/adapters/application.js

```diff
@@ -11,7 +11,6 @@ export default class ApplicationAdapter extends JSONAPIAdapter {
   session;
 
   get headers() {
-    console.log(this.session);
```

Comment:
Remove this?

Reply:
Yeah, it seemed unnecessary to have the session printing to the console. I was thinking this was probably left after some some debugging.